### DISCUSSION TITLE
21：9屏幕不适

### DISCUSSION THREAD

**Discussion by bigbug-dev** [the1812/Bilibili-Evolved#1420] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/62877853/103731995-714dd080-5021-11eb-9c35-41e7963e6b80.png)
21:9的显示器在使用清爽模式时两边有较大的空余，请问可以推出相关功能进行调试吗。

**Comment by the1812** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/26504152/103851115-ef72ab80-50e3-11eb-941b-86c9a7e21a1c.png)
大概这样多一列?

**Reply by bigbug-dev** [reply]
嗯，这样就看着很舒服
